### ISSUE TITLE
Default country not accounted for?

### ISSUE THREAD

**Issue by bdmac** [daddyz/phonelib#90]
There seems to be some code paths where the configured `default_country` for phonelib is not being used? Consider the following:

```
[9] pry(main)> Phonelib.default_country
=> "US"
[10] pry(main)> number = '4035566466'
=> "4035566466"
[11] pry(main)> Phonelib.possible?(number)
=> false
[12] pry(main)> Phonelib.parse(number).possible?
=> false
[13] pry(main)> Phonelib.parse(number, Phonelib.default_country).possible?
=> true
```

I would expect these to all return true or none of them to return true. What am I missing? Do I need to manually pass the default country to parse as in the last example above for some reason?

**Comment by daddyz** [comment]
@bdmac fixed in 0.6.6. Thanks for reporting

**Comment by bdmac** [comment]
Awesome and I confirmed this now works in 0.6.6!
